src/pages/ShopingCart.page.js

```diff
@@ -22,4 +30,30 @@ export class ShopingCartPage extends BaseSwagLabPage {
     async removeCartItemById(id) {
         await this.cartItems.nth(id).locator(this.removeItemSelector).click();
     }
+
+    async getAllItemsAddedToCart() {
+        const items = this.page.locator(this.cartItemSelector);
+        const itemsCount = await items.count();
+        const allItems = [];
+        for (let i = 0; i < itemsCount; i += 1) {
+            allItems.push(
+                {
+                    name: await items.nth(i).locator(this.inventoryItemName).textContent(),
+                    description: await items.nth(i).locator(this.inventoryItemDescription).textContent(),
+                    price: await items.nth(i).locator(this.inventoryItemPrice).textContent(),
+                },
+            );
+        }
+        return allItems;
```

Comment:
duplicates this https://github.com/sonceuvikonci/playwright_tests_with_po/pull/1/files#diff-19ed92823de2eac33a7229fa1e719ef6526f8c2e3f5090da0e8103d19f168a7eR28. You can move it to some helper and add parametrization to this method. Task with a bit higher complexity 😉